Docs edits and additions after doing an initial setup (#579)
nix-community/lanzaboote: docs/how-to-guides/enable-measured-boot.md

> This limit is enforced by `systemd-pcrlock` [which currently won't create a
> policy for more than 8
> variants](https://github.com/systemd/systemd/issues/41526).

> 

Reviewer: Please keep this line

boot.initrd.systemd.enable = true;
boot.lanzaboote = {
  measuredBoot = {
    enable = true;
    pcrs = [
      0

>       3

Reviewer: 3 should also be removed

};
```

> The additional `pcrs` entries `1` and `2` might be flaky, you can try how it behaves with you hardware and add them. See [the explanation page for measured boot](../explanation/measured-boot.md).

Reviewer 2: ```suggestion
The additional `pcrs` entries `1` and `2` might be flaky. You have to try out how they behave on your hardware. See [the explanation page for measured boot](../explanation/measured-boot.md).
```
